decidim-proposals/spec/system/endorse_proposal_spec.rb

```diff
@@ -0,0 +1,124 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "Endorse Proposal", type: :system do
```

Comment:
So we had no system specs for this? 🤔 I think it's good we're adding them then! 😄